src/base58.cpp

```diff
@@ -47,11 +47,11 @@ bool DecodeBase58(const char* psz, std::vector<unsigned char>& vch)
     int size = strlen(psz) * 733 /1000 + 1; // log(58) / log(256), rounded up.
     std::vector<unsigned char> b256(size);
     // Process the characters.
-    assert(sizeof(mapBase58)/sizeof(mapBase58[0]) == 256);  // guarantee not out of range
+    static_assert(mapBase58.size() == 256, "mapBase58.size() should be 256"); // guarantee not out of range
```

Comment:
The `static_assert` is pointless now?

Reply:
Just in case, after all `static_assert()` is no harm.